src/org/thoughtcrime/securesms/loki/LokiPublicChatPoller.kt

```diff
@@ -246,6 +246,8 @@ class LokiPublicChatPoller(private val context: Context, private val group: Loki
             displayNameUpdatees = displayNameUpdatees.union(newDisplayNameUpdatees)
         }.successBackground { messages ->
             // Process messages in the background
+            val key = group.displayName.replace(Regex("\\s*"), "")
```

Comment:
```suggestion
            val key = GroupUtil.getEncodedId(group.id.toByteArray(), false)
```
We should use addresses since they'll always exists, where as names could be `null`